src/components/form/FormInput.tsx

```diff
@@ -0,0 +1,34 @@
+import { Field } from 'formik';
+import { Fragment } from 'react';
+
+const FormInput = ({
+  label,
+  name,
+  placeholder,
+  type,
+  icon,
+  value,
+}: {
+  label?: string;
+  name: string;
+  placeholder: string;
+  type: string;
+  value?: string;
+  icon?: object;
+}) => {
+  return (
```

Comment:
additional props:
- overwriteClassName
- className -> ```const inputClasses = overrwriteClassName ? className : `my-3 h-[44px] w-full rounded-lg pl-2 bg-neutral-50 border-neutral-400 border-[1px] drop-shadow-lg active:border-blue-500 focus:border-blue-500 disabled:bg-neutral-100 disabled:text-neutral-400 invalid:border-brown-500 text-neutral-900 required ${className}\`;``` (also determine whatever padding is needed internally to account for icons)

Reply:
error strings